plugins/infinispan100/src/main/java/org/radargun/service/Infinispan100ServerService.java

```diff
@@ -42,6 +42,15 @@
    @Property(doc="A full path to a logging configuration file. If set, the server will use the provided logging configuration file. If not set, the default server logging configuration is used.")
    private String loggingConfig;
 
+   @Property(doc="Boolean property for enabling user creation on server. If set to true, the new user will be added with provided username and password. Default is false.")
+   private boolean addUser=false;
```

Comment:
Since ispn10 and jdg8.0 authentication is not mandatory, shouldn't those modifications be included only on jdg81 or infinispan11 plugin?

Reply:
@gustavolira as there is no Infinispan110ServerService.java class, and the server service is extended from jdg80 plugin, I thought to add it here and added addUser property so that if jdg80 plugin is used it is false by default and no need to add username/password to it.